mantidproject/mantid
# Workspace Manipulation / ADS docs

**Description of work.**

**Report to:** Pas 

**To test:**

* Build docs and check that the code and links work. (some code lines won't work in isolation)
* Should there be an image, say of the Workspaces toolbox?
* Where would it be useful to link to this page?

Fixes #27892 

*This does not require release notes* because **it is documentation**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
